doc/user/pim.rst

```diff
@@ -71,6 +71,39 @@ PIM Routers
    prefix of group ranges covered. This command is vrf aware, to configure for
    a vrf, specify the vrf in the router pim block.
 
+.. clicmd:: embedded-rp
```

Comment:
Since this is only valid for PIM6, move this to the pimv6.rst user doc instead.

Reply:
Moved to `pimv6.rst`.